src/core_write.cpp

```diff
@@ -152,6 +152,8 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry)
 {
     entry.pushKV("txid", tx.GetHash().GetHex());
     entry.pushKV("version", tx.nVersion);
+    entry.pushKV("size", (int)::GetSerializeSize(tx, SER_NETWORK, PROTOCOL_VERSION));
+    entry.pushKV("vsize", (GetTransactionWeight(tx) + WITNESS_SCALE_FACTOR - 1) / WITNESS_SCALE_FACTOR);
```

Comment:
`GetVirtualTransactionSize()` is declared in policy.h, which isn't included in core_write.

Why is `GetVirtualTransactionSize()` in policy? Because transaction virtual size isn't a consensus rule, and is only used for mempool admittance/miner selection. On top of that,  after https://github.com/bitcoin/bitcoin/pull/8365 , virtual size has been overloaded to mean both *transaction weight*/4 and an adjusted virtual size taking account of sigops cost.

I could update core_write.cpp to include policy, but I don't think it's a good idea to include policy more widely than necessary.